Integrations/GoogleBigQuery/GoogleBigQuery.py

```diff
@@ -176,9 +184,7 @@ def test_module():
         next(results_rows_iterator)
         demisto.results("ok")
     except Exception as ex:
-        return_error("There was a problem creating a BigQuery client.\n"
```

Comment:
please post screenshot of an error returned for example

Reply:
Added a more clear message